environment.yml

```diff
@@ -28,4 +28,4 @@ dependencies:
  - scipy
  - xarray
  - shapely
- - pydap
+ - pydap=3.2.1
```

Comment:
@leiyangleon do we want to force a single version or could we do it such we have a version excluded? I recall it was an intermediate version which is problematic?

https://stackoverflow.com/questions/54847688/how-to-specify-version-ranges-in-conda-environment-yml

Reply:
@dbekaert @jlmaurer fixed this by excluding a version as @dbekaert suggested above.